compiler+runtime/test/jank/reader-macro/symbolic-value/pass-inf.jank

```diff
@@ -0,0 +1,3 @@
+(assert (infinite? ##Inf))
```

Comment:
What about some other failure cases? Off the top of my head:

* `##Infinite`
* `##INF`
* `###Inf`
* `##NAN`
* What about `-##NaN`?

Reply:
I think `-##NaN` would go down a different code path, but I can still add it if you like.